cg/resources/raredisease_bundle_filenames.yaml

```diff
@@ -0,0 +1,965 @@
+- format: cram
+  id: SAMPLEID
+  path: PATHTOCASE/alignment/SAMPLEID_sorted_md.cram
+  path_index: PATHTOCASE/alignment/SAMPLEID_sorted_md.cram.crai
+  step: alignment
+  tag: ~
```

Comment:
I'm not sure why we have all these untagged files, but they should either be removed from here or assigned a unique tag to make them usable by Hermes

Reply:
Done